configs/milvus.yaml

```diff
@@ -28,7 +28,7 @@ etcd:
     #  - "stdout" as os.Stdout,
     #  - file path to append server logs to.
     path: stdout
-    level: info # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
+    level: warn # Only supports debug, info, warn, error, panic, or fatal. Default 'info'.
```

Comment:
nit: Default 'warn'.

Reply:
Just thought the embed etcd log is a little bit too much, especially for standalone erver.
Maybe redirect to other directory?

Reply:
Sure. I added a "path" setting above, so we can redirect logs to a specific directory. For example I use "/tmp/milvus/logs" as etcd log path in embedded Milvus.

Reply:
sure, I've change the default back to info, it's a little bit of noisy